assets/css/app/components/_progress_section.scss

```diff
@@ -1,14 +0,0 @@
-// Component with progress info and bar at the bottom (used for displaying resources usage etc.)
-.progress-section {
```

Comment:
I found that `progress-section` is used in `lib/phoenix/live_dashboard/components/card_usage_component.ex` in `line number 6` and `progress` is used in `lib/phoenix/live_dashboard/components/title_bar_component.ex` as a child class of progress-section. I think that there could be a missing style. What do you think?

Reply:
Hi ✋🏽  , 

I forgot to remove those classes from the html, `resource-usage` and` progress-section` classes no longer exist and have no effect.

The class `resource-usage-total` and `resource-usage-total-value` do not belong to a component itself, but are used in 2 pages (`home` and` os_data`), so I moved  ` resource_usage.scss` with just those two classes to the utils folder.

Finally the `utils/resource_usage.scss`  file for some reason was empty, but I have already fixed it. 🔧 

@bemesa21